htsget-search/src/storage/local.rs

```diff
@@ -83,6 +83,7 @@ mod tests {
   use std::fs::{create_dir, File};
   use std::io::prelude::*;
 
+  use super::super::BytesRange;
```

Comment:
Please replace the `super::super` by a `crate::storage::...` use.

Reply:
Done!